crates/sui-indexer-alt-jsonrpc/src/api/coin.rs

```diff
@@ -90,6 +99,29 @@ struct BalanceCursor {
 
 type Cursor = BcsCursor<BalanceCursor>;
 
+impl Coins {
+    pub fn new(
+        context: Context,
+        fullnode_rpc_url: url::Url,
+        config: WriteConfig,
```

Comment:
nit: no longer a `WriteConfig` -- you could call this a `NodeConfig`, I guess?

Reply:
Maybe it would be good to add a function on such a `NodeConfig` that returns a `Client` that can be used by both the Write API and the Coins API?